src/napari_threedee/utils/geometry.py

```diff
@@ -96,3 +96,26 @@ def point_in_layer_bounding_box(point, layer):
         return False
     else:
         return True
+
+def clamp_point_to_layer_bounding_box(point: np.ndarray, layer):
```

Comment:
I like this function! Can we move this to `/utils/napari.py`? I prefer it there since it requires napari stuff. Maybe we have a

Reply:
I put it here because of the function above which also takes a layer:
https://github.com/napari-threedee/napari-threedee/blob/8fbe1631d3f16c90e33560ed053d5ea2045c94fb/src/napari_threedee/utils/geometry.py#L92-L98

It does make some sense to move both, right?

Reply:
it does split up the bounding_box only function which is pretty generic though docstring refers to layer extent from the layer function. Could add a docstring for point_in_layer_bounding_box to point out the other function?

Reply:
Good point. Let's move both and add a note to the docstring. Thanks!!